ontology/yaml/resources/HVAC/entity_types/VAV.yaml

```diff
@@ -523,6 +523,13 @@ VAV_HWZTC_CHWZTC_DTM:
     - CHWZTC
     - DTM
 
+VAV_MDSCV:
```

Comment:
Does this equipment measure anything else in the space? For example, since the dampers are controlling based on detected motion (and you've noted it is for ventilation) I would expect `CO2C` on this type as well. Lastly, please confirm if this VAV performs zone temperature control.

Reply:
This VAV performs neither CO2 control or temperature control. It only open the damper when there is someone in the room and closed it when the room is empty.

Reply:
ok thank you for confirming